Ramos_HwAssignments/Ramos_Wk4_WeatherApp/WeatherApp.tsx

```diff
@@ -0,0 +1,57 @@
+import { Text, View } from 'react-native';
+import { NavigationContainer, useNavigation, useRoute, RouteProp, StackActions } from '@react-navigation/native';
+import { createStackNavigator } from '@react-navigation/stack';
+import { createDrawerNavigator } from '@react-navigation/drawer';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import { FontAwesomeIcon } from '@fortawesome/react-native-fontawesome';
+import { faCircle } from '@fortawesome/free-regular-svg-icons'
+import { UseFetch } from './Functions/useFetch';
+import { Weather } from './Weather';
+import { Forecast } from './Forecast';
+
+export type StackParamList = {
+  Weather: undefined;
+  Forecast: undefined;
+};
+
+const Drawer = createDrawerNavigator();
+const Tab = createBottomTabNavigator();
+
+const ForecastTab = () => {
+  return (
+    <Tab.Navigator
+      screenOptions={({ route }) => ({
+        tabBarIcon: ({ focused, color, size }) => {
+          let iconName: any;
+
+          if (route.name === 'Forecast5Day') {
+            iconName = '5-circle';
```

Comment:
I haven't tried to use FontAwesomeIcon before. I'd stick to https://icons.expo.fyi/Index for icons. It should have everything you would need